z4_1_2022.py

```diff
@@ -0,0 +1,22 @@
+def main():
+    filepath = "przyklad.txt"
+    ans = int(0)
+    rem = []
+
+
+    f = open(filepath, "r")
+    #4_1: ile liczb z pierwszą i ostatnią cyfrą taką samą oraz pierwsza z tycx liczb zapisana
+
+    for line in f:
+        #print(line[len(line) - 2])
+        if line(0) == line[len(line) - 2]:
```

Comment:
Dokładniej to problem znajduje się w tej linijce, masz taki error:
```py
Traceback (most recent call last):
  File "/home/kamil/Dev/personal/cool-invention/z4_1_2022.py", line 22, in <module>
    main()
  File "/home/kamil/Dev/personal/cool-invention/z4_1_2022.py", line 12, in main
    if line(0) == line[len(line) - 2]:
TypeError: 'str' object is not callable
```
Generalnie line to jest string, wydaje mi się że zamiast zrobić ```line[0]```, zrobiłaś ```line(0)```; Python potraktował to jako funkcję, bo tylko dla funkcji przekazujemy argumenty w okrągłych nawiasach, ale jako że jest stringiem to wywalił errora;